openmetadata-ui/src/main/resources/ui/webpack.config.dev.js

```diff
@@ -19,6 +19,7 @@ const process = require('process');
 
 const outputPath = path.join(__dirname, 'build');
 const subPath = process.env.APP_SUB_PATH ?? '';
+const devServerTarget = process.env.DEV_SERVER_TARGET ?? 'http://localhost:8585/';
```

Comment:
I am not sure if this is a good name for the variable. If you have a better name, feel free to change it.